src/lib/worker/config.ts

```diff
@@ -161,7 +161,7 @@ interface CuttleKeycapKey extends CuttleBaseKey {
 }
 
 interface CuttleBasicKey extends CuttleBaseKey {
-  type: 'ec11' | 'oled-128x32-0.91in-adafruit' | 'oled-128x32-0.91in-dfrobot' | 'evqwgd001'
+  type: 'ec11' | 'oled-128x32-0.91in-adafruit' | 'oled-128x32-0.91in-dfrobot' | 'evqwgd001' | 'adafruit-mini-thumbstick'
```

Comment:
For the name, what do you think about adding `joystick-` in front? I think it would make sense to group all the joysticks (including thumbsticks) together by name: `joystick-adafruit-mini-thumbstick` or perhaps `joystick-joycon-adafruit` (since Adafruit markets it as feeling similar to joycons).

Reply:
Changed to `joystick-adafruit-joycon`

Reply:
Could you do put the `adafruit` at the end? I've been consistently putting brands at the end since they're usually the least important term in making a mental image of the part. For example:
- `joystick`: very clear description
- `joycon`: specifying which type of joystick. If you've used a joycon before you have a clearer mental image of this joystick
- `adafruit`: only really helpful if you're trying to buy or look up the part. Unless you're intimately familiar with Adafruit's product catalog, reading `adafruit` isn't going to give you a clearer idea of what the joystick looks like.

Reply:
Changed